content/doc/book/blueocean/creating-pipelines.adoc

```diff
@@ -15,13 +15,13 @@ title: Creating Pipelines
 Blue Ocean makes it easy to create Pipelines in Jenkins.
 Pipelines can be created from existing `Jenkinsfile`s
 or from new `Jenkinsfile`s created with the
-<<pipeline-editor, Blue Ocean Pipeline Editor>>.
+<<pipeline-editor#, Blue Ocean Pipeline Editor>>.
```

Comment:
This will now link to the sibling page `../pipeline-editor/`.  Before it attempted to go to the `#pipeline-editor` anchor on this page.